package.json

```diff
@@ -37,16 +37,15 @@
   "homepage": "https://github.com/ipfs/interface-ipfs-core#readme",
   "dependencies": {
     "async": "^2.6.2",
-    "bl": "^3.0.0",
     "bs58": "^4.0.1",
-    "callbackify": "^1.1.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "cids": "~0.7.1",
     "concat-stream": "^2.0.0",
     "delay": "^4.3.0",
     "dirty-chai": "^2.0.1",
     "es6-promisify": "^6.0.2",
```

Comment:
Still needed?

Reply:
Yes, it's being used [here](https://github.com/ipfs/interface-js-ipfs-core/blob/chore/convert-tests-async-await/src/object/utils.js).